### PR TITLE
[DO NOT MERGE!] WIP on GC proposal

### PR DESCRIPTION
**Pull request by binji**

### CODE REVIEW

**Comment by binji** [comment]
@jakobkummerow PTAL! Here is some initial work to start parsing GC types.

File: src/opcode.h [hunk 1]
```diff
@@ -84,13 +84,15 @@ struct Opcode {
   Address GetAlignment(Address alignment) const;
 
   static bool IsPrefixByte(uint8_t byte) {
-    return byte == kMathPrefix || byte == kThreadsPrefix || byte == kSimdPrefix;
+    return byte == kGcPrefix | byte == kMathPrefix || byte == kThreadsPrefix ||
+           byte == kSimdPrefix;
```
Comment (Horcrux7) [inline src/opcode.h]: Look like a missing pipe.

Comment (binji) [reply src/opcode.h]: Whoops, thanks!

**Comment by keithw** [comment]
Closing as stale.

**Pull request closed without merging**